script.js

```diff
@@ -0,0 +1,462 @@
+            rusShiftButtonPic.classList.add('rusShiftButPic');
+            let rusShiftButton = document.createElement('span');
+            rusShiftButton.classList.add('rusShiftBut');
+            rusShiftButton.classList.add('hidden');
+            rusShiftButton.classList.add(`${buttonsCode[i]}`)
+            this.properties.ru == 'true' ? buttonImg.classList.add('hidden') : rusButtonImg.classList.add('hidden');
+            this.properties.ru == 'true' ? shiftButtonPic.classList.add('hidden') : rusShiftButtonPic.classList.add('hidden');
+            buttonImg.insertAdjacentText('afterbegin', buttons[i]);
+            rusButtonImg.insertAdjacentText('afterbegin', rusButtons[i]);
+            shiftButton.insertAdjacentText('afterbegin', shiftButtons[i]);
+            rusShiftButton.insertAdjacentText('afterbegin', rusShiftButtons[i]);
+            shiftButtonPic.insertAdjacentText('afterbegin', shiftButtonsPic[i]);
+            rusShiftButtonPic.insertAdjacentText('afterbegin', rusShiftButtonsPic[i]);
+            i++;
+            let paragraph = document.createElement('p');
+            paragraph.insertAdjacentText('beforeend', 'Клавиатура создана в операционной системе Windows. Для переключения языка комбинация: левыe shift + alt ');
+            buttonElement.appendChild(buttonImg);
+            buttonElement.appendChild(rusButtonImg);
+            buttonElement.appendChild(shiftButton);
+            buttonElement.appendChild(rusShiftButton);
+            buttonElement.appendChild(shiftButtonPic);
+            buttonElement.appendChild(rusShiftButtonPic);
+
+
+            switch(button) {
+                case "backspace":
+                    buttonElement.classList.add('backspace');
+                    buttonElement.addEventListener('click', () => {
+                        this.properties.value = this.properties.value.substring(0, this.properties.value.length - 1);
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
+                    });
+                    break;
+
+                    case "\\":
+                    buttonElement.classList.add('backslash');
+                    buttonElement.addEventListener('click', () => {
+                        if (this.properties.ru == 'false') {
+                            this.properties.capsLock == false ? this.properties.value += buttonElement.textContent.substring(0,1).toLowerCase() : this.properties.value += buttonElement.textContent.substring(0,1);
+                        }
+                        else if (this.properties.ru == 'true') {
+                            this.properties.capsLock == false ? this.properties.value += buttonElement.textContent.substring(1,2).toLowerCase() : this.properties.value += buttonElement.textContent.substring(1,2);
+                        }        
+                        document.getElementsByTagName ('textarea')[0].value = this.properties.value;
```

Comment:
but if you have several textareas on the page, will [0] be the correct one?

Reply:
Probably not. i'll try use more Id's in the future.